neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -35,15 +35,15 @@
         raise ModuleNotFoundError(msg)  # pylint:disable=undefined-variable
 
 try:
-    # neptune-client=0.9.0 package structure
+    # neptune-client=0.9.0+ package structure
```

Comment:
If you think it's important, remember tofix it here: https://github.com/neptune-ai/neptune-integration-template/blob/master/neptune_integration_template/impl/__init__.py

Reply:
I'm going to apply it along with other integrations.